src/scripts/main.js

```diff
@@ -1,3 +1,58 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+document.querySelector('.remove-row').addEventListener('click', () => {
+  if (table.rows.length <= 10) {
+    document.querySelector('.append-row').removeAttribute('disabled');
+  }
+
+  if (table.rows.length === 3) {
+    document.querySelector('.remove-row').setAttribute('disabled', '');
+  }
+  table.deleteRow(2);
+});
+
+document.querySelector('.append-row').addEventListener('click', () => {
+  if (table.rows.length >= 9) {
+    document.querySelector('.append-row').setAttribute('disabled', '');
+  }
+
+  if (table.rows.length === 2) {
+    document.querySelector('.remove-row').removeAttribute('disabled');
+  }
```

Comment:
it's better not use `magic` numbers such as 9, 2, 10, 3 as you applied at your code. Create variables for them